modules/nw-sriov-configuring-device.adoc

```diff
@@ -32,6 +32,7 @@ It might take several minutes for a configuration change to apply.
 * You have installed the SR-IOV Network Operator.
 * You have enough available nodes in your cluster to handle the evicted workload from drained nodes.
 * You have not selected any control plane nodes for SR-IOV network device configuration.
+* Optional. Label the `SriovNetworkNodePolicy` cluster nodes. For more information about labeling nodes, see "Understanding how to update labels on nodes". 
```

Comment:
@stevsmit Thanks for the PR!
Do you think we could be specific on what labels to add in the cluster nodes? for example:
label the SR-IOV capable cluster nodes with SriovNetworkNodePolicy.Spec.NodeSelector if not already labeled?